src/brevitas/optim/sign_sgd.py

```diff
@@ -0,0 +1,489 @@
+"""
```

Comment:
Why this license?

Reply:
Similar to https://github.com/Xilinx/brevitas/blob/master/src/brevitas/nn/quant_mha.py.